src/org/thoughtcrime/securesms/FullMsgActivity.java

```diff
@@ -115,17 +118,24 @@ public boolean onOptionsItemSelected(MenuItem item) {
         // would be required as well - probably as the leftmost button which is not that usable in
         // not-always-mode where the dialog is used more often. Or [Ok] would mean "Once" as well as "Change checkbox setting",
         // which is also a bit weird. Anyway, let's give the three buttons a try :)
-        final String checkmarkPrefix = DynamicTheme.getCheckmarkEmoji(this) + " ";
-        if (Prefs.getAlwaysLoadRemoteContent(this)) {
-          builder.setNeutralButton(checkmarkPrefix + getString(R.string.always), (dialog, which) -> onChangeLoadRemoteContent(LoadRemoteContent.ALWAYS));
-          builder.setNegativeButton(R.string.never, (dialog, which) -> onChangeLoadRemoteContent(LoadRemoteContent.NEVER));
-          builder.setPositiveButton(R.string.once, (dialog, which) -> onChangeLoadRemoteContent(LoadRemoteContent.ONCE));
+        final String checkmark = DynamicTheme.getCheckmarkEmoji(this) + " ";
+        String alwaysCheckmark = "";
+        String onceCheckmark = "";
+        String neverCheckmark = "";
+        if (!isContactRequest && Prefs.getAlwaysLoadRemoteContent(this)) {
+          alwaysCheckmark = checkmark;
+        } else if (loadRemoteContent) {
+          onceCheckmark = checkmark;
         } else {
-          builder.setNeutralButton(R.string.always, (dialog, which) -> onChangeLoadRemoteContent(LoadRemoteContent.ALWAYS));
-          builder.setNegativeButton((loadRemoteContent? "" : checkmarkPrefix) + getString(R.string.never), (dialog, which) -> onChangeLoadRemoteContent(LoadRemoteContent.NEVER));
-          builder.setPositiveButton((loadRemoteContent? checkmarkPrefix : "") + getString(R.string.once), (dialog, which) -> onChangeLoadRemoteContent(LoadRemoteContent.ONCE));
+          neverCheckmark = checkmark;
         }
 
+        if (!isContactRequest) {
+          builder.setNeutralButton(alwaysCheckmark + getString(R.string.always), (dialog, which) -> onChangeLoadRemoteContent(LoadRemoteContent.ALWAYS));
```

Comment:
(side comment: I wonder if this manual concatenation of translated text with a checkmark at the beginning is not wrong for LTR languages)

Reply:
yeah, it is maybe not perfect, but nowadays UIs have checkbox/switches left/right and no one really cares :)

it is as if the checkbox is on the right in english text, would not be worse, imho.

we can change that if there are complains :)